en/flight_modes/takeoff.md

```diff
@@ -4,7 +4,7 @@
 
 The *Takeoff* flight mode causes the vehicle to take off to a specified height and wait for further input.
 
-> **Note** 
+> **Note**

```

Comment:
FYI, all the changes to this doc are cosmetic (i.e. ignore). Just added to remind me it might be worth adding a comment here. This is waiting on confirmation of vehicles and takeoff types for which the attitude checks are run.